extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -153,32 +97,52 @@ export class PerFolderServerInstance implements IServerInstance {
 		await this.startInternal();
 	}
 
+	public spawn(command: string, args?: ReadonlyArray<string>, options?: SpawnOptions): ChildProcess {
+		return spawn(command, args, options);
+	}
+
+	public ensureProcessEnded(childProcess: ChildProcess): void {
```

Comment:
Why is this part of the class? It doesn't need access to any class members so should just be a helper function somewhere I'd say.